FlaskApp/.flaskenv.template

```diff
@@ -0,0 +1,3 @@
+FLASK_APP=__init__.py
+FLASK_DEBUG="1"
+sql_alchemy_uri = "UsernameAndPassword"
```

Comment:
This is misleading, this should tell the general template of the url
postgresql://postgres:postgres@localhost/postgres